get-jira-keys/src/index.ts

```diff
@@ -46,33 +68,102 @@ async function getJiraKeysFromPr() {
 async function getJiraKeysFromPush() {
     try {
         console.log("Get Jira keys from Push");
-        const payload = github.context.payload;
+
         let resultArr: any = [];
 
         payload.commits.forEach((commit: any) => {
             let matchedKeys = matchKeys(commit.message);
             if (matchedKeys) {
-                resultArr.push(matchedKeys);
+                resultArr = resultArr.concat(matchedKeys);
             }
         });
 
+        resultArr = resultArr?.filter(onlyUnique);
+
         return resultArr.join(',');
     } catch (error) {
         core.setFailed(error.message);
     }
 }
 
-async function run(): Promise<void> {
+async function getJiraKeysFromRelease() {
+    try {
+        console.log("Get Jira keys for latest release");
+
+        //Commits number contains 'release/x.x.x' message
+        const releaseMsgNum = 1; 
+
+        let resultArr: any = [];
+        let commitsArr: any = [];
```

Comment:
1. Do not put data type literals inside the variable name (`resultArr` -> `jiraKeys`, `commitsArr` -> `commits`); 
2. Its good if you use strong types.

Reply:
fixed